doc/source/whatsnew/v1.1.0.rst

```diff
@@ -289,6 +289,9 @@ Other enhancements
 - Make :class:`pandas.core.window.Rolling` and :class:`pandas.core.window.Expanding` iterable（:issue:`11704`)
 - Make ``option_context`` a :class:`contextlib.ContextDecorator`, which allows it to be used as a decorator over an entire function (:issue:`34253`).
 - :meth:`groupby.transform` now allows ``func`` to be ``pad``, ``backfill`` and ``cumcount`` (:issue:`31269`).
+- :meth:`Series.plot` now supports asymmetric error bars. Previously, if :meth:`Series.plot` received a "2xN" array with
+  error values for `yerr` and/or `xerr`, the left/lower values (first row) would be mirrored, while the right/upper values (second row) are
+  ignored. Now, the first row represents the left/lower error values and the second row the right/upper error values. (:issue:`9536`)
```

Comment:
Couple of things:

1. Can you put this all on a single line? It's fine if it's long
2. Can you make the tenses agree? So,
```diff
- while the right/upper values (second row) are ignored.
+ while the right/upper values (second row) were ignored.
```

Reply:
Updated in https://github.com/pandas-dev/pandas/pull/34514/commits/fa4915b7e39251b6ad6d19cdd6355009f94bce80